includes/wikia/services/UserStatsService.class.php

```diff
@@ -55,23 +49,23 @@ public function increaseEditsCount() {
 		global $wgMemc;
 
 		if ( !$this->validateUser() ) {
-			return false;
+			return;
```

Comment:
This is a general remark: for the sake of better code, we should avoid `null` as much as possible: nullchecks, returning nulls, etc. This method has side-effect, therefore it should return the status of it - whether it worked or not. Or, it should generally return the new number of edits and throw and exception on failure.